packages/solid-form/package.json

```diff
@@ -22,7 +22,7 @@
     "test:lib": "vitest",
     "test:lib:dev": "pnpm run test:lib --watch",
     "test:build": "publint --strict",
-    "build": "vite build"
+    "build": "vite build && tsc -p tsconfig.build.json"
```

Comment:
Leaving a comment here to remind @lachlancollins to investigate moving this into our Vite build if at all possible

Reply:
@crutchcorn I've tweaked the config slightly so that it generates a flatter structure. AFAIK, there is no need for the `dist/source` folder to have its own types, since they are identical to those in `dist/esm` (but if not, it can be re-enabled by removing `"declaration": false`).

I agree it might be nice to have a direct TS->JS copy of the source built by `@tanstack/config`, I'll have a look!